runtime/vm/ContinuationHelpers.cpp

```diff
@@ -148,15 +148,16 @@ createContinuation(J9VMThread *currentThread, j9object_t continuationObject)
 }
 
 j9object_t
-synchronizeWithConcurrentGCScan(J9VMThread *currentThread, j9object_t continuationObject, ContinuationState volatile *continuationStatePtr)
+synchronizeWithConcurrentGCScan(J9VMThread *currentThread, j9object_t continuationObject, volatile ContinuationState *continuationStatePtr)
 {
-	ContinuationState oldContinuationState = 0;
-	ContinuationState returnContinuationState = 0;
+	volatile ContinuationState oldContinuationState = 0;
+	volatile ContinuationState returnContinuationState = 0;
+	volatile ContinuationState newContinuationState = 0;
```

Comment:
I'd remove volatile here, and in other methods where used for local variables (not for a pointer to a volatile location)